app/controllers/group_members_controller.rb

```diff
@@ -0,0 +1,72 @@
+class GroupMembersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    group = Group.find(params[:group_id])
+
+    res = current_user &&
+      (group.is_admin?(current_user) || group.is_mod?(current_user))
```

Comment:
We could also express it as `group.member(current_user).admin?` but it'd probably be slower since AR would have to deserialize and marshal a whole Member object while (iirc) `is_admin?(current_user)` uses an `exists?` call